src/gui/folderman.cpp

```diff
@@ -37,8 +37,20 @@
 #include <QNetworkProxy>
 
 namespace {
-const char versionC[] = "version";
-const int maxFoldersVersion = 1;
+/*
+ * [Accounts]
+ * 0\version=1
+ */
+auto versionC()
```

Comment:
Isn't this a duplicate of the one in `folder.cpp`?

Reply:
Both are version but in different section that are evaluated independently.
But yes...
However refactoring the settings stuff is stuff for a major release...